api_v3/lib/types/KalturaWidget.php

```diff
@@ -115,13 +134,20 @@ public function getMapBetweenObjects ( )
 		return array_merge ( parent::getMapBetweenObjects() , self::$map_between_objects );
 	}
 
-	public function toWidget () 
+	public function toInsertableWidget ()
 	{
-		$user = new widget();
+		$object = new widget();
```

Comment:
I think $widget is the best attr name here same goes for toUpdatableWidget function